packages/inputs-minimal/src/password_field/index.stories.jsx

```diff
@@ -29,4 +30,43 @@ export const Default = () => {
   );
 };
 
+export const LoginForm = () => {
+  const [textFieldValue, setTextfieldValue] = useState('');
```

Comment:
Why not using formik for consistency?

Reply:
To me, this feels more like an unnecessary extension of the code here. 
We're actually don't need any of the formik features like special validation, schemas, onSubmit or something like that. 
This should rather demonstrate that using the browsers passwords managers inside a `<form>` with `<input>`s won't break the UI.